novawallet/Info.plist

```diff
@@ -71,6 +81,10 @@
 		<string>PublicSans-ExtraLight.otf</string>
 		<string>PublicSans-Medium.otf</string>
 	</array>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>remote-notification</string>
```

Comment:
add this because of it's required for didReceiveRemoteNotification:fetchCompletionHandler: